.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,9 +5,9 @@ To ease the process of reviewing your PR, do make sure to complete the following
 
 - [ ] Write a good description on what the PR does.
 - [ ] Create a news fragment with `towncrier create <IssueNumber>.<type>` which will be
-  included in the changelog. `<type>` can be one of: breaking, user_action, new_check, removed_check, extension,
-  false_positive, false_negative, bugfix, other, internal. If necessary you can write
-  details or offer examples on how the new change is supposed to work.
+  included in the changelog. `<type>` can be one of: breaking, user_action, feature,
+  new_check, removed_check, extension, false_positive, false_negative, bugfix, other, internal.
```

Comment:
This section is also in [contribute.rst](https://github.com/PyCQA/pylint/blob/main/doc/development_guide/contributor_guide/contribute.rst), so we'd need to add this there as well.